config/formats.js

```diff
@@ -226,6 +226,34 @@ exports.Formats = [
+			const megaStone = ["Abomasite", "Absolite", "Aerodactylite", "Aggronite", "Alakazite", "Altarianite", "Ampharosite", "Audinite", "Banettite", "Beedrillite", "Blastoisenite", "Blazikenite", "Cameruptite", "Charizardite X", "Charizardite Y", "Diancite", "Galladite", "Garchompite", "Gardevoirite", "Gengarite", "Glalitite", "Gyaradosite", "Heracronite", "Houndoominite", "Kangaskhanite", "Latiasite", "Latiosite", "Lopunnite", "Lucarionite", "Manecite", "Mawilite", "Medichamite", "Metagrossite", "Meteorite", "Mewtwonite X", "Mewtwonite Y", "Pidgeotite", "Pinsirite", "Sablenite", "Salamencite", "Sceptilite", "Scizorite", "Sharpedonite", "Slowbronite", "Steelixite", "Swampertite", "Tyranitarite", "Venusaurite"];
+			const zCrystal = ["Aloraichium Z", "Buginium Z", "Darkinium Z", "Decidium Z", "Dragonium Z", "Eevium Z", "Electrium Z", "Fairium Z", "Fightinium Z", "Firium Z", "Flyinium Z", "Ghostium Z", "Grassium Z", "Groundium Z", "Icium Z", "Incinium Z", "Kommonium Z", "Lunalium Z", "Lycanium Z", "Marshadium Z", "Mewnium Z", "Mimikium Z", "Normalium Z", "Pikanium Z", "Pikashunium Z", "Poisonium Z", "Primarium Z", "Psychium Z", "Rockium Z", "Snorlium Z", "Solganium Z", "Steelium Z", "Tapunium Z", "Ultranecrozium Z", "Waterium Z"];
+			const template = this.getTemplate(set.species || set.name);
+			if (!sinnohDex.includes(template.baseSpecies) && !sinnohDex.includes(template.species) && template.num > 251) {
```

Comment:
You only need the first argument but replace template.baseSpecies with set.species

Reply:
This means you don’t need to define the template at all